src/main/java/com/faforever/client/news/NewsController.java

```diff
@@ -108,38 +87,14 @@ public void onUnreadNewsEvent(UnreadNewsEvent unreadNewsEvent) {
 
 
   private void loadNews() {
-    newsService.fetchNews().thenAccept(newsItems -> {
-      Platform.runLater(() -> {
-        newsListView.getItems().setAll(newsItems);
-        onLoadingStop();
-        if (!newsItems.isEmpty()) {
-          NewsItem mostRecentItem = newsItems.get(0);
-          preferencesService.getPreferences().getNews().setLastReadNewsUrl(mostRecentItem.getLink());
-          preferencesService.storeInBackground();
-        }
-        newsListView.getSelectionModel().selectFirst();
-      });
+    Platform.runLater(() -> {
+      newsWebView.getEngine().load("https://www.faforever.com/news");
```

Comment:
Never ever do hardcoded urls. 
We either load it from the FafApiProperties class and declare it in the application.yml
Or we add it to the remote preferences which would give us a bit more flexibility until things have settled.

Reply:
I can do it like the Unitdb, that uses the application.yml and Clientproperties.java
Is that a good way?